addons/l10n_es_edi_tbai/data/template_invoice.xml

```diff
@@ -106,7 +106,8 @@
                 </t>
             </CabeceraFactura>
             <DatosFactura t-if="is_emission">
-                <FechaOperacion t-out="format_date(invoice.invoice_date) if format_date(datetime_now) != format_date(invoice.invoice_date) else None"/>
+                <FechaOperacion t-if="invoice.delivery_date" t-out="format_date(invoice.delivery_date) if format_date(datetime_now) != format_date(invoice.delivery_date) else None"/>
```

Comment:
According to 
> The “operation date” field must be completed when the date of completion of the operation is different from the date of issue of the invoice.

Shouldn't it be like this instead?
```suggestion
                <FechaOperacion t-if="invoice.delivery_date and format_date(invoice.invoice_date) != format_date(invoice.delivery_date)" t-out="format_date(invoice.delivery_date)"/>
```

Reply:
the date of issue of the invoice is the date when the scheduled action is run so no ;-)

Reply:
maybe @Olivier-olma could confirm?

Reply:
@alialfie according to chkl `FechaOperacion is delivery date if set, invoice_date otherwise` so I believe we should do like it I proposed

Reply:
I think that we need to check delivery date != invoice date and if yes fill it

Reply:
I really think this is not sufficient, let me explain myself, according to the note on the ticket, `The “operation date” field must be completed when the date of completion of the operation is different from the date of issue of the invoice.` it makes me believe that we have to :
1) check if delivery_date is set, in order to compare it emission date, if both are equal then we don't complete the field, if not we fill it.
2) If no delivery date is set, then we do the same for invoice_date, invoice_date being different from emission date in some cases